src/main/target/RCEXPLORERF3/target.c

```diff
@@ -26,10 +26,10 @@
 #include "drivers/dma.h"
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
-    DEF_TIM(TIM3,  CH2, PA4, TIM_USE_MOTOR, 1), // PWM1 - PA4
-    DEF_TIM(TIM17, CH1, PA7, TIM_USE_MOTOR, 1), // PWM2 - PA7
     DEF_TIM(TIM1,  CH1, PA8, TIM_USE_MOTOR, 1), // PWM3 - PA8
+    DEF_TIM(TIM17, CH1, PA7, TIM_USE_MOTOR, 1), // PWM2 - PA7
```

Comment:
Not a biggie, but could probably do with a renumbering in the comments, to avoid future confusion.